utility-libraries/custom_lib_example/Dockerfile

```diff
@@ -0,0 +1,23 @@
+ARG ARCH=armv7hf
+ARG VERSION=3.4.2
+ARG UBUNTU_VERSION=20.04
+ARG REPO=axisecp
+ARG SDK=acap-sdk
```

Comment:
This is acap-native-sdk, please control the rest of the example too. Update of VERSION is also required.